pkg/cli/plugin_cmd.go

```diff
@@ -164,6 +164,7 @@ func getCmdForPluginEx(p *PluginInfo, cmdName string, mapEntry *plugin.CommandMa
 
 		runner := NewRunner(p.Name, p.InstallationPath, completion)
 		ctx := context.Background()
+		setupPluginEnv(srcHierarchy, dstHierarchy)
```

Comment:
This one does not have any impact at the moment since shell completion does not use any of those environment variables.  However, to be future-proof, I felt adding it was the right thing to do.